src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/AttributeValueResolver.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Controller\ArgumentResolver;
+
+use Psr\Container\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Attribute\ValueResolver;
+use Symfony\Component\HttpKernel\Controller\ValueResolverInterface;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
+
+/**
+ * Delegates value resolving of arguments bearing the {@see ValueResolver} attribute to the specified resolver.
+ *
+ * @author Mathieu Lechat <mathieu.lechat@les-tilleuls.com>
+ */
+final class AttributeValueResolver implements ValueResolverInterface
```

Comment:
I suggest naming it `SpecifiedResolverValueResolver` instead, to better describe what it does (`AttributeValueResolver` is easily confused with `RequestAttributeValueResolver` that resolved a value from attributes)